src/main/java/com/okta/tools/OktaAwsCliEnvironment.java

```diff
@@ -0,0 +1,12 @@
+package com.okta.tools;
+
+public class OktaAwsCliEnvironment {
+    public static String oktaOrg;
+    public static String oktaUsername;
+    public static String oktaPassword;
+    public static String oktaProfile;
+
+    public static String oktaAwsAppUrl;
+
+    public static String awsRoleToAssume;
+}
```

Comment:
:art: These are mutable global variables.

It would be preferable for each of the individual classes to take a **OktaAwsCliEnvironment** as a constructor argument and have it as a field and then uses of each of the classes are constructed and interacted with as instances.